SpatialGDK/Source/SpatialGDKEditor/Private/Utils/LaunchConfigEditor.cpp

```diff
@@ -25,6 +53,11 @@ void ULaunchConfigurationEditor::SaveConfiguration()
 
 	if (bSaved && Filenames.Num() > 0)
 	{
-		GenerateDefaultLaunchConfig(Filenames[0], &LaunchConfiguration);
+		if (ValidateGeneratedLaunchConfig(LaunchConfiguration, LaunchConfiguration.ServerWorkersMap))
+		{
+			GenerateDefaultLaunchConfig(Filenames[0], &LaunchConfiguration, LaunchConfiguration.ServerWorkersMap);
+
+			OnConfigurationSaved.ExecuteIfBound(this, Filenames[0]);
+		}
```

Comment:
If this failed we should log an error

Reply:
There seem to be enough log inside in case of failure in the called function.